lib/testing/src/lib/process-services-cloud/pages/edit-process-filter-cloud-component.page.ts

```diff
@@ -96,6 +96,10 @@ export class EditProcessFilterCloudComponentPage {
         await browser.sleep(1500);
     }
 
+    async setStartedBy(option: string): Promise<void> {
```

Comment:
After yesterday meeting i understood we shouldn't add these kind of methods anymore.